Lib/calendar.py

```diff
@@ -37,6 +41,35 @@ def __str__(self):
         return "bad weekday number %r; must be 0 (Monday) to 6 (Sunday)" % self.weekday
 
 
+# Constants for months referenced later
+@global_enum
+class Month(IntEnum):
+    JANUARY = 1
+    FEBRUARY = 2
+    MARCH = 3
+    APRIL = 4
+    MAY = 5
+    JUNE = 6
+    JULY = 7
+    AUGUST = 8
+    SEPTEMBER = 9
+    OCTOBER = 10
+    NOVEMEMBER = 11
+    DECEMBER = 12
+
+
+# Constants for weekdays
+@global_enum
+class Weekday(IntEnum):
```

Comment:
Weekday sometimes means a day other than Saturday or Sunday. 
```suggestion
class Day(IntEnum):
```